lib/views/after_auth_screens/events/create_event_page.dart

```diff
@@ -183,18 +186,36 @@ class _CreateEventPageState extends State<CreateEventPage> {
                         final date = await customDatePicker(
                           initialDate: model.eventEndDate,
                         );
-                        setState(() {
-                          model.eventEndDate = date;
-                        });
+                        final startDate = model.eventStartDate;
+                        if (startDate.compareTo(date) < 0) {
+                          setState(() {
+                            model.eventEndDate = date;
+                          });
+                        } else {
+                          // ignore: undefined_method
+                          navigationServiceLocal.showTalawaErrorWidget(
+                            "End Date cannot be after start date ",
+                          );
+                        }
                       },
                       setTime: () async {
                         final time = await customTimePicker(
                           initialTime: model.eventEndTime,
                         );
-
-                        setState(() {
-                          model.eventEndTime = time;
-                        });
+                        final currTimeToInt = time.hour + time.minute / 60.0;
+                        final startTime = model.eventStartTime;
+                        final startTimeToInt =
+                            startTime.hour + startTime.minute / 60;
+                        if (startTimeToInt.compareTo(currTimeToInt) < 0) {
```

Comment:
Just double checking this
would this stop a user from setting event time to 5:30pm on Tuesday when it's currently 6:30pm on Sunday?

Reply:
Thanks for the review, 
You're right. Now I've added that restriction and also the check at the start date to make sure that start date is not any date before today's date.
Please take a look  : https://github.com/PalisadoesFoundation/talawa/pull/1616/commits/b7f368748c8ef9c53ee2b68f311dda7ff2fdba38

Reply:
Now I've tried all the combination of inputs and it is giving correct result with proper date and time. ✔️